GooglePlayInstant/Editor/AppBundlePublisher.cs

```diff
@@ -66,14 +73,24 @@ public static bool Build(string aabFilePath)
         {
             bool buildResult;
             Debug.LogFormat("Building app bundle: {0}", aabFilePath);
-#if UNITY_2018_4_OR_NEWER
+            // As of February 2019, every released version of Unity natively supporting AAB has used Android Gradle
+            // Plugin 3.2.0 which includes bundletool 0.5.0. Bundletool 0.6.0+ is needed for uncompressNativeLibraries
+            // and version 0.6.1+ is needed for uncompressNativeLibraries with instant apps.
+            // One can #define PLAY_INSTANT_ENABLE_NATIVE_ANDROID_APP_BUNDLE to build using the native AAB builder.
+#if PLAY_INSTANT_ENABLE_NATIVE_ANDROID_APP_BUNDLE
+#if PLAY_INSTANT_HAS_NATIVE_ANDROID_APP_BUNDLE
```

Comment:
Could we indent the nested #ifs?
I know we don't normally indent preprocessor directives, but in this case, since the #if's are nested, I think it would improve readability.

Reply:
Our IDE's "reformat code" won't like it, but I agree that it would help readability. Done.